src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.diff.sql

```diff
@@ -0,0 +1,391 @@
+/*
+We are making the following changes in this version of the schema
+-- Adding two bit columns to StringSearchParam table - IsMin and IsMax
+-- Updating the corresponding table value type to include these two new values
+-- Updating the Upsert stored procedure to handle these two new values
+*/
+
+CREATE TYPE dbo.StringSearchParamTableType_2 AS TABLE
+(
+    SearchParamId smallint NOT NULL,
+    Text nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
+    TextOverflow nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL,
+    IsMin bit NOT NULL,
+    IsMax bit NOT NULL
+);
+
+ALTER TABLE dbo.StringSearchParam
+ADD IsMin bit NOT NULL,
```

Comment:
If NOT NULL you will need to add a `DEFAULT` clause.

Reply:
Updated the script to add two constraints for the two columns.

Reply:
Have you verified that the schema comparison integration test still passes>

Reply:
Whats the plan for a migration strategy, do we plan to have an update for these new columns as part of the diff script?

Reply:
I actually hadn't considered the option of updating these column values as part of the upgrade script. My original plan was to enable that via the `$reindex` mechanism. But I will spend some time investigating whether doing it via the upgrade script might be a better option.